source/_docs/scene/editor.markdown

```diff
@@ -3,23 +3,30 @@ title: "Scenes Editor"
 description: "Instructions on how to use the scenes editor."
 ---
 
-In Home Assistant 0.102 we introduced the first version of our scene editor. If you just created a new configuration with Home Assistant, then you're all set! Go to the UI and enjoy.
+In Home Assistant 0.102 we introduced the first version of our {% term scene %} editor. If you just created a new configuration with Home Assistant, then you're all set! Go to the UI and enjoy.
```

Comment:
Let's remove the versioning stuff here, it isn't really relevant to tell that (especially considering that is like 3+ years ago 😓 )